lib/lsperfm.rb

```diff
@@ -1,21 +1,37 @@
 require 'yaml'
 require 'lsperfm/core'
+require 'terminal-table'
 
 module LogStash
   module PerformanceMeter
 
     extend self
 
     def invoke
-      debug   = !!ENV['DEBUG']
-      headers = !!ENV['HEADERS']
+      debug           = 'false' != ENV.fetch('DEBUG', 'false')
+      display_headers = 'false' != ENV.fetch('HEADERS', 'true')
+      table_output    = 'false' != ENV.fetch('TABLE_OUTPUT', 'false')
 
-      install_path  = ARGV.size > 1 ? ARGV[1] : Dir.pwd
-      definition    = ARGV.size > 0 ? ARGV[0] : ""
+      install_path    = ARGV.size > 1 ? ARGV[1] : Dir.pwd
+      definition      = ARGV.size > 0 ? ARGV[0] : ""
 
       runner = LogStash::PerformanceMeter::Core.new(definition, install_path)
       runner.config = '.lsperfm' if File.exist?('.lsperfm.yml')
-      puts runner.run(debug, headers).join("\n")
+
+      results = runner.run(debug)
```

Comment:
Makes the runner/core return all it's info - headers & results; letting this method deal with how to display it.